typescript/packages/subsurface-viewer/src/storybook/layers/Grid3DLayer.stories.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Suggest not using this globally, but instead for each instance you want to disable.

Reply:
Ooops, fixed.